lib/string-checker.js

```diff
@@ -130,7 +130,12 @@ StringChecker.prototype = {
 
         this._configuredRules.forEach(function(rule) {
             errors.setCurrentRule(rule.getOptionName());
-            rule.check(file, errors);
+
+            try {
```

Comment:
Time lower = Speed higher ;-)

V8 magic or something. Not sure how...

I suggest to talk about speed in #1043 since we all know that we need it and there are a lot of things we can speed up. If we've got troubles with perfomance because of this try-catch statement we could move it out into tryCatch helper function (as well as in bluebird) and I don't see here a problem.